dbms/src/Functions/FunctionsRound.cpp

```diff
@@ -10,6 +10,7 @@ void registerFunctionsRound(FunctionFactory & factory)
     factory.registerFunction<FunctionFloor>("floor", FunctionFactory::CaseInsensitive);
     factory.registerFunction<FunctionCeil>("ceil", FunctionFactory::CaseInsensitive);
     factory.registerFunction<FunctionTrunc>("trunc", FunctionFactory::CaseInsensitive);
+    factory.registerFunction<FunctionRoundDown>();
```

Comment:
Why is it registered differently from other functions?

Reply:
This is Ok.
It means default registration - with case sensitive name that function provides itself.